apps/worker/src/app/workflow/usecases/execute-bridge-job/execute-bridge-job.usecase.ts

```diff
@@ -225,6 +225,7 @@ export class ExecuteBridgeJob {
             message: response.message,
             code: response.code,
             data: response.data,
+            cause: response.cause,
```

Comment:
store the root cause of the novu bridge request failure on the execution details